groundcontrol/admin-bundle.tasks.js

```diff
@@ -81,11 +82,17 @@ adminBundle.tasks.bundle = createBundleTask({
 
 adminBundle.tasks.bundleOptimized = createBundleTask({
     config: {
-        entry: adminBundle.config.srcPath + 'jsnext/app.js',
+        entry: `${adminBundle.config.srcPath}jsnext/app.js`,
         output: {
-            filename: adminBundle.config.distPath + 'js/admin-bundle.next.js',
+            filename: 'admin-bundle.next.js',
+            path: path.resolve(__dirname, '../src/Kunstmaan/AdminBundle/Resources/public/js'),
```

Comment:
Can you use the adminBundle.config here? As all other paths are in the config object